### ISSUE TITLE
Badging in Arctos?

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2111]
Sorry for filling up GitHub today - I am trying to post up ideas and information from SPNHC before I forget and get back into the weeds of daily work.

@amgunderson referred to Arctos with the phrase "choose your own adventure" during his pubs and projects demo camp. It is true that this is how Arctos works! The during the AWG meeting (which was really a potential Arctos users Q&A) we ended up using the term "Arctos Super User" to describe Aren.

All of this got me to thinking about badging and whether it might be nice to have Arctos users be able to earn badges which could then inform others who has experience in what. Not critical, but an idea for helping the community see who has the most experience when there is a question about something.

**Comment by dustymc** [comment]
Dibs on "Data Janitor!"

**Comment by campmlc** [comment]
could badges be emojis?
We could have a mammal, a bird, a plant, a fossil . . .
Data emoji?

**Comment by Jegelewicz** [comment]
> Dibs on "Data Janitor!"

Oh there should be levels, but yours would be Steven Segal "Taking out the garbage"

**Comment by Jegelewicz** [comment]
> could badges be emojis? We could have a mammal, a bird, a plant, a fossil . . . Data emoji?

I was thinking more along the lines of 

Bulkloader Badge
Novice (bulkloaded <10 records)
Advanced Beginner (has bulkloaded between 10 and 99 records)
Competent (has bulkloaded between 100 and 499 records)
Proficient (has bulkloaded between 500 and 2000 records)
Expert (has bulkloaded more than 2000 records)

Note: levels come from https://www.health.nsw.gov.au/nursing/projects/Documents/novice-expert-benner.pdf

It would be nice if badges would be awarded automatically based upon Arctos metrics....

**Comment by Jegelewicz** [comment]
But yes, the badges should be emoji-like:

![image](https://user-images.githubusercontent.com/5725767/58925569-4690ed00-8705-11e9-9b41-36a355e94ae3.png)

![image](https://user-images.githubusercontent.com/5725767/58925600-56103600-8705-11e9-9e44-4d2b946d925e.png)

**Comment by Jegelewicz** [comment]
Here is where we should start for badges - https://badgr.org/

**Comment by ewommack** [comment]
I feel like this might be related to Agent rankings: https://github.com/ArctosDB/newsletter/issues/70

> Here is where we should start for badges - https://badgr.org/

Does that mean we'd want to share badges across platforms? Maybe with GloBI?

And I support @dustymc for Data Janitor

**Comment by Jegelewicz** [comment]
> Does that mean we'd want to share badges across platforms? Maybe with GloBI?

YES! I definitely thought about that - collaborative badging would be GREAT!
